contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -111,6 +117,9 @@ def span_data_to_envelope(self, sd):
                 data.type = 'INPROC'
         # TODO: links, tracestate, tags
         for key in sd.attributes:
+            # This removes redundant data from AppInsights
+            if 'http.' in key:
```

Comment:
I'm not sure about this one, we do have users who expect **all** the attributes to show up in the custom key-value-pair in Kusto, following the native format described in OpenCensus specification.

Reply:
Another option would be to remove them form the dictionary if they are already used inside the span attributes. Would that be ok?

Reply:
I'm on a fence now. Removing it would give us benefit of less junk/duplicated traffic. Keeping it would make it easier for people to go to the key-value pair and find all the OpenCensus attributes according to the spec.

Reply:
Maybe include in the docs that if people want duplicate information, they could add it in the span attributes with another tags? Or add a `debug` flag in the exporter that includes everything?

Reply:
I tend to be very conservative about introducing configurations/flags as they increase the complexity.
As I don't have strong opinion on this, feel free to remove these as duplicates.

BTW, I would use `key.startswith('http.')`.

Reply:
> I tend to be very conservative about introducing configurations/flags as they increase the complexity.
Not my favourite option also.

> BTW, I would use key.startswith('http.')
Agreed and fixed!